api/v2beta1/helmrelease_types.go

```diff
@@ -286,6 +287,14 @@ type HelmChartTemplateSpec struct {
 	// +optional
 	// +deprecated
 	ValuesFile string `json:"valuesFile,omitempty"`
+
+	// Verify contains the secret name containing the trusted public keys
+	// used to verify the signature and specifies which provider to use to check
+	// whether OCI image is authentic.
+	// This field is only supported for OCI sources.
+	// Chart dependencies, which are not bundled in the umbrella chart artifact, are not verified.
+	// +optional
+	Verify *sourcev1.OCIRepositoryVerification `json:"verify,omitempty"`
```

Comment:
This pulls in the source-controller API into our own API, which I would like to avoid.

Reply:
Let's copy paste that struct for now. We can move it to `apis` like we did for `kustomize` to avoid pulling SC API into Helm API.